data/core/common.lua

```diff
@@ -240,6 +359,16 @@ function common.draw_text(font, color, text, align, x,y,w,h)
 end
 
 
+---Runs benchmark on a function.
+---
+---When the benchmark is completed, the execution time and
+---percentage of frame time for the benchmark is printed.
+---The frame rate is always assumed to be 60 FPS, thus
+---a value of 100% would mean that the benchmark took
+---1/60 of a second to execute.
```

Comment:
Mmm, dunno if calling this a benchmark is appropriate, considering that it only executes the function once.
Maybe something like `Times the execution of a function...`.